src/styles/blocks/page.scss

```diff
@@ -0,0 +1,19 @@
+.page {
+  margin: 0;
+  padding-top: 0;
+}
+
+.page-body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.container {
```

Comment:
these styles shouldn't be here, since these elements are not elements of `page` block (1 file per one BEM-block)